tests/conftest.py

```diff
@@ -1,3 +1,5 @@
+import csv
```

Comment:
I would reassess where you `autouse=True` for these fixtures. You don't really want to autouse the cli runner, for example, as that's only needed for cli tests. I would only use autouse for fixtures that are truly universal, and otherwise pass the specific fixtures you need to the given test functions.